logs/error.log

```diff
@@ -0,0 +1,4 @@
+2019-03-18 21:56:03,935 WARNING: 404 Error Resource Not Found [in /Users/tresor/Desktop/SolarSENSE/SolarSENSE/app/routes.py:315]
```

Comment:
We don't need your logs. Can you add the logs folder to the `.gitignore` please?